packages/react-components/react-charts-preview/library/src/components/CommonComponents/ResponsiveContainer/useResponsiveContainerStyles.styles.ts

```diff
@@ -0,0 +1,33 @@
+import { makeStyles, mergeClasses } from '@griffel/react';
+import { ResponsiveContainerStyles } from './ResponsiveContainer.types';
+
+export const responsiveContainerClassNames: ResponsiveContainerStyles = {
+  root: 'fui-charts-resp__root',
+};
+
+const useStyles = makeStyles({
+  root: {
+    width: '100%',
+    height: '100%',
+
+    '& [class*="chartWrapper"]': {
+      width: '100%', // optional
+      // To prevent chart height from collapsing while resizing
+      height: '100%', // optional
+    },
+
+    '& svg': {
+      // This overrides the pixel width and height of svg allowing it to resize properly within flexbox
+      width: '100%',
+      height: '100%',
+    },
+  },
+});
+
+export const useResponsiveContainerStyles_unstable = (): ResponsiveContainerStyles => {
```

Comment:
Consider passing component state to this hook even if you don't use it today.  This will help if you ever need to adjust styles based on props/state.

Reply:
The standard contract for style hooks in Fluent is `useComponentStyles = (state: ComponentState): ComponentState`

So this function should also return component state. [Here's an example](https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-button/library/src/components/Button/useButtonStyles.styles.ts#L539).

Reply:
The responsive container is not meant to have a style component. It is an invisible container.
Still added the prop to it to follow the standard approach.